projects/admin/src/app/circulation/main-request/requested-items-list/requested-items-list.component.html

```diff
@@ -0,0 +1,44 @@
+<!--
+ RERO ILS UI
+ Copyright (C) 2019 RERO
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, version 3 of the License.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<div class="row mb-1 py-2 d-flex align-items-center bg-dark text-white rounded">
+  <div class="col-2" translate>Item</div>
+  <div class="col-5" translate>Document</div>
+  <div class="col-2" translate>Call number</div>
+  <div class="col-2" translate>Request date</div>
+</div>
+
+<ng-container *ngIf="items">
+  <ng-container *ngIf="items.length > 0 else noitems">
+    <admin-requested-item
+      *ngFor="let item of items"
+      [item]="item"
+      [isCollapsed]="isCollapsed"
+      [callout]="getItemCallout(item)"
+      (requestValidated)="validateRequest($event)"
+    ></admin-requested-item>
+  </ng-container>
+  <ng-template #noitems>
+    {{ 'no request to validate' | translate }}
+  </ng-template>
+</ng-container>
+
+<ng-container *ngIf="items === null">
```

Comment:
Same comment as previous about the sipnner.

Reply:
Difficult to use ngxSpinnerService in the compoenent because `items` provide from parent component. 
Maybe use the SpinnerService in this parent component.